### ISSUE TITLE
Cant filter bees and honneycombs by type

### ISSUE THREAD

**Issue by reneberwanger** [sayterdarkwynd/FrackinUniverse#1496]
**Situation:**

I make and huge item network, here i get all stuff from apiarys and send to be processed on Industrial Centrifuge. But i need to filter only honeycombs to go there.

Currently, I´m using 5 ITDs to do that, because i can only use "exact" method to do that.

**Possible Solution / Sugestion**

Change the type or category from all bee stuff. Everything now is "crafting material". I´m looking into code, and i found this file, but i dont understand what represents the property name, only the category.

In my case, i found "queen":"bee" and "drone":"bee"

My suggestion is make all that as different kinds for item filter.
If you tell me how this file works, I can provide a pull request.

https://github.com/sayterdarkwynd/FrackinUniverse/blob/master/scripts/kheAA/transferconfig.config

**Comment by sayterdarkwynd** [comment]
It will remain as craftingMaterial, because that's where I want them to be placed in the players inventory. This is pretty much non-negotiable.

the properties you mention are for the bees to work with it, basically. Nothing we can do *there* for the ITD. 

I'm open to other solutions, but its not really a big problem IMO. I'll mention it to the dev that handles that device and see what we can do.

**Comment by reneberwanger** [comment]
@sayterdarkwynd But how it works the transferconfig.config ? For me its clear that is a "sub category" system just for "ITD". My suggestion only aplly to ITD "sub category" system. The bees and honeycombs will stay untouched.

**Comment by Kherae** [comment]
Hi. I'm khe, the ITDs are mine. You don't really understand how the system works, so i'll explain it:
the CATEGORY defined in the item's file is what the ITD sees as a Type. ITD Categories are groupings of those Types. IE: cosmetic leg armor is its own type, has its own category defined in its file, but it is in the Category (grouping) of Cosmetic Armors. 
While I have special handling for some items, like saplings and thrown items that lack categories, I will not implement special handling for honeycombs and the like. As sayter is not willing to change their categories (I suggested they have Queen and Drone categories), nothing is going to change.
I designed my system to work based on item categories, not tags, as categories are the norm on all vanilla items. Modders sometimes forget them, but that's on them.

So...nothing changing.
